protos/joystick/joystick.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
+
+package mavsdk.rpc.joystick;
```

Comment:
I would probably call it `manual_control` because it is not limited to joystick. It could also be gamepad, etc.

And you could also use a joystick with offboard control.